src/Api/Management/Controller/DeveloperController.php

```diff
@@ -95,7 +96,28 @@ public function getEntities(
     }
 
     /**
-     * @inheritdoc
+     * {@inheritdoc}
+     *
+     * We are enforcing email addresses over developer ids (UUIDs) when we are updating user profile.
+     *
+     * @psalm-suppress PossiblyNullArgument $entity->originalEmail() is not null here.
+     *
+     *  @see https://github.com/apigee/apigee-client-php/issues/153
```

Comment:
Nitpick unnecessary space before `@see`

Reply:
@mxr576 Updated.
Thanks for noticing it.